### PR TITLE
Explain static memory

### PR DESCRIPTION
**Pull request by matthiasgoergens**
It's a bit hard to figure out what we mean by 'static memory' or 'memory with static addresses' from the code.  I had to dig in the git history, but I'm not sure I guessed right.  Please check whether I guessed right.

### CODE REVIEW

File: ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs [hunk 1]
```diff
@@ -17,6 +17,8 @@ pub struct MmuConfig<E: ExtensionField> {
     /// Initialization of registers.
     pub reg_config: <RegTableCircuit<E> as TableCircuit<E>>::TableConfig,
     /// Initialization of memory with static addresses.
+    ///
+    /// This is a generalisation of 'program data'.
```
[the remark this answers is not in the document]
Comment (naure) [reply ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs:21]: Also best documented on the type itself.

[the remark this answers is not in the document]
Comment (matthiasgoergens) [reply ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs:21]: Thanks.  Could you add that documentation where you think it fits best, please?

[the remark this answers is not in the document]
Comment (kunxian-xia) [reply ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs:21]: Are the explanations in https://github.com/scroll-tech/ceno/pull/609 sufficient for understanding?

[the remark this answers is not in the document]
Comment (matthiasgoergens) [reply ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs:21]: Nice, thanks!

**Comment by matthiasgoergens** [comment]
https://github.com/scroll-tech/ceno/pull/609 did the job better than I could.  Thanks!

**Pull request closed without merging**